src/main/webapp/app/entities/submission.model.ts

```diff
@@ -89,6 +91,12 @@ export function getFirstResult(submission: Submission | undefined): Result | und
     }
 }
 
+export function refreshLatestResult(submissions: Submission[]) {
```

Comment:
```suggestion
export function setLatestResult(submissions: Submission[]) {
```

Reply:
refresh sounds like retrieving it from the server, also make sure to add JavaDoc style documentation

Reply:
another thing: I would also offer this for one submission, then you don't need to wrap submissions in array when calling this method. As there is no overloading for Typescript functions, you need to use two different names